app/web/src/components/Navigation/Navigation.tsx

```diff
@@ -241,17 +393,47 @@ export default function Navigation() {
             </div>
           </Hidden>
         </div>
-        <div className={classes.bug}>
-          <Hidden smDown>
-            <ExternalNavButton
-              route={handbookURL}
-              label={HELP}
-              labelVariant="h3"
-            />
-            <NavButton route={logoutRoute} label={LOG_OUT} />
-          </Hidden>
-          <ReportButton />
-        </div>
+
+        <Hidden smDown>
+          <div ref={menuRef} style={{ display: "flex", flexDirection: "row" }}>
+            <div
+              style={{ marginRight: 16, display: "flex", alignItems: "center" }}
+            >
+              <ReportButton />
+            </div>
+            <div
+              className={classes.menuBtn}
+              onClick={menuOpen ? handleMenuClose : handleMenuOpen}
```

Comment:
```suggestion
              onClick={() => { setMenuOpen(!menuOpen) }}
```
Could inline and simplify to this rather than create two separate event handlers, since this is only toggling the boolean and so the abstraction seems unnecessary.